src_test/simple_transcriptor.hpp

```diff
@@ -156,9 +161,7 @@ struct simple_transcriptor_with_nonconst_interface :
         this->update(first, last);
         this->out() << "))";
         this->set_in_value(false);
-        if constexpr (!std::is_const_v<Ch>) {
-            *last = Ch();
-        }
+        *last = Ch();
```

Comment:
The spec says on `finalize` of a `TableHandler` object:
> After evaluating, the parser shall not dereference the range [bb, l] before the next evaluation of (5). [ Note: Therefore the table handler may freely modify the element in the range if the reference-removed type of *f and *l are not const-qualified. —end note ]

So it seems that this branching is not required.